src/components/member-card/index.js

```diff
@@ -7,16 +7,32 @@ import PropTypes from 'prop-types';
 import ShowSkills from '@components/member-card/show-skills';
 import SuperUserOptions from '@components/member-card/super-user-options';
 
-const Card = ({ developerInfo, isOptionKey }) => {
+const Card = ({ developerInfo, isOptionKey, colorCombination }) => {
   const { query } = useRouter() || { query: { dev: false } };
   const { dev } = query;
   const { username, first_name, last_name, img_url, isMember } = developerInfo;
+  const { color_primary, color_secondary } = colorCombination;
+  const cardRef = useRef();
+  const textRef = useRef();
```

Comment:
Why are we using refs here? Can we not derive this data that's used in the component?

Reply:
removed the refs